inc/commondbtm.class.php

```diff
@@ -3414,16 +3417,17 @@ static function getCompleteNameField() {
    }
 
 
-   /** Get raw name of the object
-    * Maybe overloaded
+   /**
+    * Deprecated, replaced by getFriendlyName
     *
+    * @deprecated
     * @see CommonDBTM::getNameField
-    *
     * @since 0.85
     *
     * @return string
    **/
    function getRawName() {
+      \Toolbox::deprecated();
 
       if (isset($this->fields[static::getNameField()])) {
```

Comment:
You should call `$this->getFriendlyName()` here, so removing overrides of `getRawName` will be transparent for end user.